### DISCUSSION TITLE
I can't uninstall bthps3

### DISCUSSION THREAD

**Discussion by ChangoD123** [nefarius/BthPS3#95] [Q&A]
I installed this driver and now my Bluetooth adapter stopped working (error code 10) and I can't uninstall it without Bluetooth, any solution?

**Comment by nefarius** [comment]
Solution is covered in the FAQ.

**Reply by ChangoD123** [reply]
Sorry, I've read and I can't find a solution. Could it affect that my Bluetooth adapter is 4.0?
